pkg/controller/gcp/database/operations.go

```diff
@@ -41,7 +41,7 @@ type localOperations interface {
 	addFinalizer(context.Context) error
 	isReclaimDelete() bool
 	isInstanceReady() bool
-	needUpdate(*sqladmin.DatabaseInstance) bool
+	needUpdates(*sqladmin.DatabaseInstance) bool
```

Comment:
Sorry, by "s" I meant `needsUpdate` not `needUpdates`, as in "this needs an update".